app/src/main/java/fr/neamar/kiss/SettingsActivity.java

```diff
@@ -396,6 +397,37 @@ public void onIncluded(final @NonNull AppPojo app) {
         category.addPreference(excludedAppsScreen);
     }
 
+    private void addExcludedShortcutAppSettings() {
+        final DataHandler dataHandler = KissApplication.getApplication(this).getDataHandler();
+
+        PreferenceScreen excludedAppsScreen = ExcludePreferenceScreen.getInstance(
+                this,
+                new ExcludePreferenceScreen.IsExcludedCallback() {
+                    @Override
+                    public boolean isExcluded(@NonNull AppPojo app) {
+                        return app.isExcludedShortcuts();
+                    }
+                },
+                new ExcludePreferenceScreen.OnExcludedListener() {
+                    @Override
+                    public void onExcluded(final @NonNull AppPojo app) {
+                        dataHandler.addToExcludedShortcutApps(app);
+                    }
+
+                    @Override
+                    public void onIncluded(final @NonNull AppPojo app) {
+                        dataHandler.removeFromExcludedShortcutApps(app);
+                    }
+                },
+                R.string.ui_excluded_from_shortcuts_apps,
+                R.string.ui_excluded_apps_dialog_title
+        );
+
+        PreferenceGroup category = (PreferenceGroup) findPreference("search-providers");
```

Comment:
IMO this is not right place for new preference.
All other settings for exclusions currently can be found on `History settings` sceen within `exclude_apps_category` group, which may be better suited (if title of group would be renamed to something more generic like "excluded entries")

Reply:
Done!